src/aws_lc_rs_algs.rs

```diff
@@ -29,6 +29,12 @@ impl SignatureVerificationAlgorithm for AwsLcRsAlgorithm {
         message: &[u8],
         signature: &[u8],
     ) -> Result<(), InvalidSignature> {
+        match self.public_key_alg_id {
+            alg_id::ECDSA_P256 | alg_id::ECDSA_P384 | alg_id::ECDSA_P521 => {
+                ec_compressed_or_uncompressed(public_key)?
+            }
+            _ => {}
+        };
```

Comment:
nit: I'd be tempted to write this with `if(matches!(...))` to avoid the empty match arm:

```rust
        if matches!(
            self.public_key_alg_id,
            alg_id::ECDSA_P256 | alg_id::ECDSA_P384 | alg_id::ECDSA_P521
        ) {
            ec_compressed_or_uncompressed(public_key)?
        };
```